test/unit/regapic.ts

```diff
@@ -139,4 +139,80 @@ describe('regapic', () => {
       );
     });
   });
+
+  describe('should support long data type conversion in proto message', () => {
+    it('large number long data type conversion in proto message response', done => {
+      const requestObject = {name: 'shelves/shelf-name/books/book-name'};
+      const responseObject = {
+        name: 'book-name',
+        author: 'book-author',
+        title: 'book-title',
+        read: true,
+        bookId: '9007199254740992',
```

Comment:
This might be my ignorance of this testing framework, but is there a place where you set the book to be an int64 and see it encoded as a string? (Asking because all the `bookId`s here look like strings...)

Reply:
Right. I removed the quote.
Book Message with value is
```
Book {
  name: 'book-name',
  author: 'book-author',
  title: 'book-title',
  read: true,
  bookId: Long { low: 0, high: 2097152, unsigned: false }
}
```
Convert to JSON object
```
{
  name: 'book-name',
  author: 'book-author',
  title: 'book-title',
  read: true,
  bookId: '9007199254740992'
}
```